### DISCUSSION TITLE
Update URL for jenkins to HTTPS

### DISCUSSION THREAD

**Discussion by theirix** [Homebrew/discussions#794] [Everyday usage]
Now Jenkins formulae have a non-secure URL like `url "http://mirrors.jenkins.io/war-stable/2.263.4/jenkins.war"`.
If we consult the Jenkins website, the official URL for the mirror download is `https://get.jenkins.io/war-stable/2.263.4/jenkins.war`. Old URL had been introduced a long time ago. Should it be changed to the new HTTPS URL?

Same thing for non-LTS `jenkins` formula. I understand that download is checked with a checksum, but downloading from HTTP source is not a good practice.

**Comment by SMillerDev** [comment]
Please make a pull request to update this.

**Reply by theirix** [reply]
Done in https://github.com/Homebrew/homebrew-core/pull/71041
